thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss

```diff
@@ -127,12 +127,19 @@
 
 $colors:
   "blue" $te-blue,
+  "light_blue" $te-orange,
```

Comment:
is this mapping accurate?

Reply:
It's very much a hack. I want the dots to be displayed in bright color, but the description in readable, regular color. One alternative would be to hard-code a "lighten" in the anomaly graph wrapper when it encounters a baseline event (event.isBaseline == true). thoughts?

Reply:
SASS provides a lighten mixin already. 
doc: http://sass-lang.com/documentation/Sass/Script/Functions.html#lighten-instance_method
usage: https://codepen.io/baudoin/pen/HdliD

As always, come find me if you have any questions!